src/packet.proto

```diff
@@ -10,9 +27,14 @@ message packet {
     }
     packet_type type = 2;
     bytes payload = 3;
-    uint64 timestamp = 4;
+    uint64 tmst = 4; // internal timer of concentrator, only useful for reply timing
```

Comment:
Can we rename this to `internal_conc_timer`? Or `conc_timer` Or something else perhaps a bit more descriptive?